dipy/denoise/localpca.py

```diff
@@ -78,7 +79,19 @@ def localpca(arr, sigma, patch_radius=1, tou=0, rician=True):
                     # compute EVD of the covariance matrix of X get the matrices W and D, hence get matrix Y = XW
                     # Threshold matrix D and then compute X_est = YW_transpose D_est
                     [d,W] = np.linalg.eigh(C)
-                    d[d < tou[i][j][k][:]] = 0
+                    # for sigma estimate we perform the median estimation
+
+                    # find the median of the standard deviation of the eigenvalues
+                    median_sqrt = np.median(np.sqrt(d[d > alpha]))
```

Comment:
I think `alpha` needs to be a parameter of the function, otherwise you are assuming this was already set (as in your example)